src/main/java/org/jenkinsci/deprecatedusage/DeprecatedUsage.java

```diff
@@ -65,8 +93,8 @@ public void analyzeWithClassVisitor(File pluginFile, ClassVisitor aClassVisitor)
                     InputStream is = warReader.getInputStream();
                     analyze(is, aClassVisitor);
                 } catch (Exception e) {
-                    System.err.println("Failed to fully analyze " + pluginFile + ".  " + fileName + " not scanned due to -> ");
-                    e.printStackTrace();
+                    System.err.println("Failed to fully analyze " + pluginFile + ".  " + fileName + " not scanned due to " + e.getMessage());
+//                    e.printStackTrace();
```

Comment:
why commented out? spamming?

remove it if no longer needed?

```suggestion
```

Reply:
Yes it was spamming in my testing cases